app/models/rbac.rb

```diff
@@ -122,7 +120,7 @@ def self.get_self_service_objects(user_or_group, klass)
   # @return [Array<Integer>] object_ids owned by a user or group
   def self.get_self_service_object_ids(user_or_group, klass)
     targets = get_self_service_objects(user_or_group, klass)
-    targets = targets.collect(&:id) if targets.respond_to?(:collect)
+    targets = targets.reorder(nil).collect(&:id) if targets.respond_to?(:collect)
```

Comment:
targets is a scope here.
If there is an order clause and we select(:id), then it will throw a sql error. Just ignore the order - we are not using it.

Hoping to remove this collect anyway.